packages/app-store/routing-forms/pages/router/getServerSideProps.ts

```diff
@@ -130,22 +130,24 @@ export const getServerSideProps = async function getServerSideProps(
   }
 
   const decidedAction = matchingRoute.action;
-
-  const { createContext } = await import("@calcom/trpc/server/createContext");
```

Comment:
Removing tRPC context/caller overhead, instead importing the responseHandler directly.